install/infra/terraform/gke/variables.tf

```diff
@@ -0,0 +1,81 @@
+variable "project" {
+  type        = string
+  description = "The project ID to create the cluster."
+}
+
+variable "kubeconfig" {
+  type        = string
+  description = "Path to write the kubeconfig output to"
+  default     = "./kubeconfig"
+}
+
+variable "region" {
+  type        = string
+  description = "The region to create the cluster."
+  default     = "europe-west1"
+}
+
+variable "zone" {
+  type        = string
+  description = "The zone to create the cluster."
+  default     = "b"
+}
+
+variable "kubernetes_version" {
+  type        = string
+  description = "Kubernetes version to be setup"
+  default     = "1.21.11-gke.1100"
+}
+
+variable "name" {
```

Comment:
why make this configurable? Do you wan to support more than one cluster per GCP project?

Reply:
I see the possibility of parallel CI jobs creating parallel clusters; parameterizing the cluster name is cheap to do now but can be a bit painful down the road. I'm in favor of this approach.

(Side note; if we have multiple parallel CI jobs then the terraform state bucket will need to have a [prefix](https://www.terraform.io/language/settings/backends/gcs#prefix) per run, but this is a yak that we can shave another day.)

Reply:
Like @adrienthebo  suggested, the main reason for considering configurable name parameter is parallel CI runs(it is highly likely since we are planning to test multiple configs). I was hoping that having the `default` name reduces the hassle a bit.
Regarding the state @adrienthebo , this is a great point. The way I have worked around it currently, is by using [terraform workspaces](https://www.terraform.io/language/state/workspaces) in the [Makefile](https://github.com/gitpod-io/gitpod/pull/10365/files/5832150fbf31b334b333493f86a2ad5b7ef0dcd1#diff-bd7b7f3a397df93111a6b6e2a4284b01bd57e04474bc9f91f1b02ee1960399a2R28) since I couldn't set the the `prefix` using variables. Each job is run in its own tf workspace and this means the state file will have the name `<workspace-name>.tfstate`.